### PR TITLE
[WIP]  Add another perf test

### PR DESCRIPTION
**Pull request by pierDipi**
Test name: `p1-r3-unord-b1-t1-64kb`

### CODE REVIEW

File: bin/generate-test-cases.sh [hunk 1]
```diff
@@ -61,3 +61,17 @@ rm -rf tests/broker/kafka
...
   --payload-file payloads/payload.68KB.txt \
   --delivery-order unordered
+
+./bin/kafka_broker_generator.py \
+  --num-brokers 1 \
+  --num-triggers 1 \
+  --num-partitions 1 \
+  --resources-output-dir tests/broker/kafka/p1-r3-unord-b1-t1-64kb/resources \
+  --hf-output-dir tests/broker/kafka/p1-r3-unord-b1-t1-64kb \
+  --name-prefix p10-r3-unord-b1-t1-64kb \
```
Comment (aliok) [inline bin/generate-test-cases.sh:71]:
```suggestion
  --name-prefix p1-r3-unord-b1-t1-64kb \
```

**Comment by aliok** [comment]
https://github.com/openshift-knative/eventing-hyperfoil-benchmark/pull/70 is building on top of this PR

File: bin/test-kafka-broker-upstream-nightly.sh [hunk 2]
```diff
@@ -12,18 +12,5 @@ export TEST_CASE_NAMESPACE=perf-test
 #./bin/run_test.sh || exit 1
 
 kubectl delete ns "${TEST_CASE_NAMESPACE}" --ignore-not-found || exit 1
-export TEST_CASE=tests/broker/kafka/p10-r3-ord-b20-t10-32kb
-# ./bin/run_test.sh || exit 1
+export TEST_CASE=tests/broker/kafka/p1-r3-unord-b1-t1-64kb
```
[the remark this answers is not in the document]
Comment (pierDipi) [reply bin/test-kafka-broker-upstream-nightly.sh:15]: It goes over the 4h timeout, the previous runs are very close to 4h, so we cannot really add it to CI run

[the remark this answers is not in the document]
Comment (aliok) [reply bin/test-kafka-broker-upstream-nightly.sh:15]: Fair enough. Let's keep the case but not run it in CI then.
If you can fix the test scripts, let's merge this as I have #70 building on top of it.

**Comment by aliok** [comment]
@pierDipi I made some scripting changes in https://github.com/openshift-knative/eventing-hyperfoil-benchmark/pull/70 (did more than that actually) and it depends on this PR. Let's either close this one and I can include the changes in #70 , or let's merge this one.

**Comment by pierDipi** [comment]
> @pierDipi I made some scripting changes in #70 (did more than that actually) and it depends on this PR. Let's either close this one and I can include the changes in #70 , or let's merge this one.

Ok, I will make my changes here and then we can move to #70

**Comment by aliok** [comment]
> Ok, I will make my changes here and then we can move to https://github.com/openshift-knative/eventing-hyperfoil-benchmark/pull/70

SGTM

File: bin/generate-test-cases.sh (hunk 1 above)
Comment (pierDipi) [reply bin/generate-test-cases.sh]: Done

File: bin/test-kafka-broker-upstream-nightly.sh (hunk 2 above)
[the remark this answers is not in the document]
Comment (pierDipi) [reply bin/test-kafka-broker-upstream-nightly.sh]: Done

**Pull request closed without merging**